docs/DATA_PREPARATION_REALITY.md

```
@@ -204,14 +204,14 @@ Use actual source diversity when possible:
 ```python
 def calculate_entropy_better(text, metadata):
     sources = []
-    
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)